src/uucore/src/lib/features/checksum.rs

```diff
@@ -612,6 +615,9 @@ where
             return Ok(());
         }
 
+        // if any incorrectly formatted line, show it
+        cksum_output(&res, status);
+
```

Comment:
Is there a specific reason for moving this up ?

Reply:
As shown in the code snippet in #6796 :
```
$ ../../../gnu/src/cksum -c CHECKSUM --ignore-missing
../../../gnu/src/cksum: dir: Is a directory
dir: FAILED open or read
../../../gnu/src/cksum: WARNING: 1 listed file could not be read
../../../gnu/src/cksum: CHECKSUM: no file was verified
```
warnings in `cksum_output` show up first,  and `CHECKSUM: no file was verified` shows up later.

Reply:
Right, indeed :+1: